test/gov/gpstest_apireg.sh

```diff
@@ -13,7 +13,7 @@ echo -e "Pattern is set to $PATTERN"
     {
       "serviceOrgid": "IBM",
       "serviceUrl": "https://bluehorizon.network/service-gps",
-      "serviceArch": "",
+      "serviceArch": "amd64",
```

Comment:
This can be left empty by the user, so we want it to stay that way in the tests. Same goes for the change to PWS.